mainnet-011: Fjord setRecommended PV & ownership transfers (WIP) (#238)
ethereum-optimism/superchain-ops: tasks/eth/011-proto-ver+ownership-transfers/README.md

# Mainnet Recommended Protocol Version Update (7.0.0 - Fjord) & Ownership Transfers

> Status: DRAFT

Reviewer: per the list of [allowed statuses](https://github.com/ethereum-optimism/superchain-ops/blob/fc2bf8c5dff3a7643c5159565e2e1fa23e5789cc/script/utils/check-task-statuses.sh#L4)—this will cause CI to fail once this is updated against the latest main
```suggestion
Status: DRAFT, NOT READY TO SIGN
```

## Objective

This is the playbook to
- Update the recommended protocol version of the `ProtocolVersions` contract on Ethereum mainnet to 7.0.0 (Fjord). It is currently set to 6.0.0 (Ecotone).

> - Transfer ownership of the `ProtocolVersions` and `SystemConfig` from the `FoundationOperationsSafe` to the `FoundationUpgradeSafe` (TODO).

Reviewer: ```suggestion
- Transfer ownership of the `ProtocolVersions` and `SystemConfig` from the `FoundationOperationsSafe` to the `FoundationUpgradeSafe`.
```

This transaction should be sent on **Jun 27, 2024**, so right after the Veto Period for Voting Cycle 23b has ended.

> It's part of a set of transactions sent as part of the [Fjord upgrade proposal](https://gov.optimism.io/t/upgrade-proposal-9-fjord-network-upgrade/8236), which is currently being voted on.

Author: ```suggestion
It's part of a set of transactions sent as part of the [Fjord upgrade proposal](https://gov.optimism.io/t/upgrade-proposal-9-fjord-network-upgrade/8236), which is currently going through the veto period.
```

> It's part of a set of transactions sent as part of the [Fjord upgrade proposal](https://gov.optimism.io/t/upgrade-proposal-9-fjord-network-upgrade/8236), which is currently going through the veto period.

Reviewer: Same comment as https://github.com/ethereum-optimism/superchain-ops/pull/232#discussion_r1648182103 to match our standard checklist format somewhere in this section

Author: done, although without reference to the equivalent Sepolia task, because we didn't bump the PVs on Sepolia (due to time constraints, it's an optional signal and not critical).
